.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+    // NOTE (aseipp): don't autoformat {markdown, toml, et cetera} files, since
+    // there's no real standardized format we've ever tried to agree on; until
+    // then, such a feature tends to make diffs much worse and various people's
+    // editors config may have some "helpful" plugins that autoformat these
+    // filetypes, so this disables a footgun for everyone.
+    "[markdown]": {
```

Comment:
I'm not very happy about this. For one, this is incompatible with somebody having another per-workspace config for their `jj` repo.

I wonder whether VSCode provides some sort of "suggested config", so this'd work similarly to the `envrc.recommended`.

Reply:
I'm not sure, BTW, whether this problem is theoretical or practical: do people often use `.vscode/settings.json` for something else? So, if this matters to you enough (and the sort-of-solution from my other comment doesn't work), we could still go with it.